### PR TITLE
#161678559 Send a confirmation email upon successful Account Verification

### PR DESCRIPTION
**Pull request by Sulaiman-Mozes**
#### What does this PR do?
Sends a confirmation email upon successful account verification

#### Description of Task to be completed?
 - A user receives an email upon successful account Verification
 - The email is to inform the user that the account has been verified 

#### How should this be manually tested?
 - An email is sent to a user upon successful verification 

#### What are the relevant pivotal tracker stories?
[#161678559]( https://www.pivotaltracker.com/story/show/161678559)

### CODE REVIEW

File: authors/apps/authentication/views.py [hunk 1]
```diff
@@ -129,6 +129,14 @@ def get(self, request, uidb64, activation_token):
...
             user.is_verified = True
             user.save()
+            self.mail_subject = "Account Verification Confirmation."
+            self.message = """
+                Hi {},
+                Your Author Haven Account has been successfully verified.
+                Thank you
+                """.format(user.username)
+            SendEmail.send_email(self, self.mail_subject, self.message,
+                                 user.email)
```
Comment (judeinno) [inline authors/apps/authentication/views.py:139]: Great you used the Send email lib, I think the chore has been fully done

**Review by judeinno — APPROVED** [review]

**Review by emugaya — CHANGES_REQUESTED** [review]
Fix hound comments

**Review by brucemakallan — APPROVED** [review]
Works fine

**Review by dnuwa — APPROVED** [review]

**Review by RachaelNantale — APPROVED** [review]
LGTM

**Review by emugaya — APPROVED** [review]

**Pull request merged**